doc/reference/misc.md

```diff
@@ -27,424 +27,1169 @@ ret
 :::
 -->
 
-## Buffered channels.
+## Buffered channels
 ::: tip To use the bindings from this module:
 ``` scheme
 (import :std/misc/channel)
 ```
 :::
 
 ### make-channel
-::: tip usage
-```
-(make-channel ...)
+``` scheme
+(make-channel [n = #f]) -> channel
+
+  n := optional fixnum, number of values channel buffer can hold
 ```
-:::
 
-Please document me!
+Creates a new buffered channel, a synchronization construct useful for sending
+and receiving data between producers and consumers, implicitly locking when
+reading from or writing to the channel. Chaining multiple channels, one after
+another, allows building computation pipelines with ease. *n* specifies how many
+elements the channel buffer is allowed to hold before blocking, with `#f` never
+blocking at all.
 
-### channel?
-::: tip usage
-```
-(channel? ...)
+:: tip Examples:
```

Comment:
argh, this is missing a `:` -- it needs 3.

Reply:
Damn, don't know how that happend.